src/test/java/uk/gov/hmcts/ccd/BaseTest.java

```diff
@@ -189,11 +189,11 @@ public void clearDownData() {
         return jdbcTemplate.queryForList("SELECT * FROM pg_catalog.pg_tables").stream()
             .filter(tableInfo -> tableInfo.get("schemaname").equals("public"))
             .map(tableInfo -> (String)tableInfo.get("tablename"))
-            .filter(BaseTest::notLiquibase)
+            .filter(BaseTest::notFlyaway)
             .collect(Collectors.toList());
     }
 
-    private static boolean notLiquibase(String tableName) {
+    private static boolean notFlyaway(String tableName) {
         return !tableName.equals("databasechangelog") && !tableName.equals("databasechangeloglock");
```

Comment:
Probably need to update to this or similar:
```suggestion
    private static boolean notFlyway(String tableName) {
        return !tableName.equals("flyway_schema_history");
```